src/events/ResultPreviewsManagerEvents.ts

```diff
@@ -1,5 +1,15 @@
 import { ISearchResultPreview } from '../magicbox/ResultPreviewsManager';
 
+/**
+ * Executed when a {@link Suggestion} is focused before {@link PopulateSearchResultPreviews} is called to fetch more options.
+ */
+export interface IUpdateResultPreviewsManagerOptionsEventArgs {
+  /**
+   * How many milliseconds should a {@link Suggestion} be focused for before {@link PopulateSearchResultPreviews} is called.
+   */
```

Comment:
We should mention the default of `200` ms in the description.

Reply:
I'll add it.